examples/README.md

```diff
@@ -0,0 +1,125 @@
+# go-qemu examples
```

Comment:
Mind breaking the long lines in this file to ~80 characters each?

Makes it easier to edit for those of us who use `vim` and such.